frontend/src/features/authz/authzSlice.js

```diff
@@ -451,19 +472,31 @@ export const authzSlice = createSlice({
       })
       .addCase(getGrantsToMe.fulfilled, (state, action) => {
         const chainID = action.payload?.chainID || "";
-        if (chainID.length) {
+        const { changeAuthzTab } = action.meta?.arg;
+
+        if (chainID?.length) {
+          let grants = action.payload.data.grants;
+          if (changeAuthzTab) {
```

Comment:
Do we need this condition? `changeAuthzTab` is always true.

Reply:
https://github.com/vitwit/resolute/blob/c7d36a1576a556477cd763f0f1b2e5f9f0925397/frontend/src/components/CustomAppBar.js#L59-L65C1

only the getGrantsToMe called in customAppBar.js have changeAuthzTab=true

The grantsToMe calls should't effect AuthzTabPanel

Reply:
Removing the condition shouldn't be a problem (though having it reduces the unnecessary AuthzTabPanel updates)